Screens/Profile.tsx

```diff
@@ -0,0 +1,293 @@
+                                    onBlur={handleBlur("name")}
+                                    onChangeText={handleChange("name")}
+                                    autoCapitalize={"words"}
+                                    value={values.name}
+                                />
+
+                                {errors.name ? (
+                                    <Text style={styles.errors}>
+                                        {errors.name}
+                                    </Text>
+                                ) : null}
+
+                                <Text style={styles.label}> Email</Text>
+
+                                <TextInput
+                                    style={styles.input}
+                                    onBlur={handleBlur("email")}
+                                    onChangeText={handleChange("email")}
+                                    keyboardType={"email-address"}
+                                    value={values.email}
+                                />
+
+                                {errors.email ? (
+                                    <Text style={styles.errors}>
+                                        {errors.email}
+                                    </Text>
+                                ) : null}
+
+                                <Text style={styles.label}> Phone</Text>
+
+                                <TextInput
+                                    style={styles.input}
+                                    onBlur={handleBlur("phone")}
+                                    keyboardType={"phone-pad"}
+                                    onChangeText={handleChange("phone")}
+                                    value={values.phone}
+                                />
+
+                                {errors.phone ? (
+                                    <Text style={styles.errors}>
+                                        {errors.phone}
+                                    </Text>
+                                ) : null}
+
+                                <View style={styles.buttons}>
+                                    <CustomButton
+                                        title="Cancel"
+                                        buttonWidth={75}
+                                        buttonOnPress={() => {
+                                            setModalVisible(!modalVisible);
+                                        }}
+                                    />
+
+                                    <CustomButton
+                                        title="Done"
+                                        isDisabled={
+                                            Object.keys(errors).length > 0
+                                        }
+                                        buttonWidth={75}
+                                        buttonOnPress={() => {
+                                            setModalVisible(!modalVisible);
```

Comment:
you could add some kind of "Are you sure?" Alert here before you submit if you wanted

Reply:
cancel button satisfies this